### PR TITLE
wr-190: Show real prompt data on article view

### PR DESCRIPTION
**Pull request by kovAnya**

### CODE REVIEW

File: frontend/src/pages/article/article-page.tsx [hunk 1]
```diff
@@ -40,24 +40,18 @@ const ArticlePage: React.FC = () => {
...
     <Loader isLoading={isLoading} hasOverlay type="circular">
       <Layout>
         <div className={styles.articlePageWrapper}>
-          <ArticleView
-            tags={MOCKED_TAGS}
-            text={article?.text ?? ''}
-            title={article?.title ?? ''}
-            coverUrl={article?.coverUrl ?? ''}
-          />
+          {getArticleStatus === DataStatus.FULFILLED && (
```
Comment (IrynaK-a) [inline frontend/src/pages/article/article-page.tsx]: It seems redundant. We've already done this check earlier (Loader and getArticleStatus === DataStatus.REJECTED)

File: frontend/src/pages/article/components/article-view/styles.module.scss [hunk 2]
```diff
@@ -58,33 +58,31 @@
...
+  flex-grow: 1;
+  padding: 18px 33px 62px;
+}
+
 .text {
-  padding: 24px 124px 62px 121px;
+  max-width: 586px;
+  margin-top: 28px;
+  margin-right: auto;
+  margin-left: auto;
```
Comment (IrynaK-a) [inline frontend/src/pages/article/components/article-view/styles.module.scss]: You can simplify this: margin: 28px auto 0;

File: frontend/src/pages/article/shared-article-page.tsx [hunk 3]
```diff
@@ -41,25 +41,18 @@ const SharedArticlePage: React.FC = () => {
...
       <Layout>
         <div className={styles.articlePageWrapper}>
-          <ArticleView
-            tags={MOCKED_TAGS}
-            text={article?.text ?? ''}
-            title={article?.title ?? ''}
-            coverUrl={article?.coverUrl ?? ''}
-            isShared
-          />
+          {dataStatus === DataStatus.FULFILLED && (
```
Comment (IrynaK-a) [inline frontend/src/pages/article/shared-article-page.tsx]: It seems redundant too.

File: frontend/src/libs/types/tag.type.ts [hunk 4]
```diff
@@ -1,6 +1,10 @@
+import { type PromptCategory } from 'shared/build/packages/prompts/prompts.js';
```
Comment (andanf-e) [inline frontend/src/libs/types/tag.type.ts]:
```suggestion
import { PromptCategory } from '~/packages/prompts/libs/enums/enums.js';
```

File: frontend/src/pages/article/libs/helpers/get-article-view-tags.helper.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { getArticleTags } from '~/libs/helpers/helpers.js';
+import { type TagType } from '~/libs/types/types.js';
+import { type ArticleResponseDto } from '~/packages/articles/articles.js';
+
+const getArticleViewTags = (article: ArticleResponseDto): TagType[] | null => {
+  const tags = getArticleTags(article);
+  if (tags.length > 0) {
```
Comment (andanf-e) [inline frontend/src/pages/article/libs/helpers/get-article-view-tags.helper.ts]:
```suggestion
  if (tags.length) {
```

File: frontend/src/pages/article/libs/helpers/get-article-view-tags.helper.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+    return tags.filter((tag) => tag.category !== 'genre');
```
Comment (andanf-e) [inline frontend/src/pages/article/libs/helpers/get-article-view-tags.helper.ts]:
```suggestion
    return tags.filter((tag) => tag.category !== PromptCategory.GENRE);
```

**Review by andanf-e — CHANGES_REQUESTED** [review]

File: frontend/src/libs/types/tag.type.ts (hunk 4 above)
Comment (kovAnya) [reply frontend/src/libs/types/tag.type.ts]: I've changed path, but kept type declaration, eslint insists on it.

**Review by andanf-e — APPROVED** [review]

**Review by mhrechkin — APPROVED** [review]

**Pull request merged**